concepticondata/conceptlists.tsv

```diff
@@ -276,4 +276,5 @@ Nordlinger-2019-57	Nordlinger, Rachel and Thieberger, Nick	2019		57	basic	Englis
 Ritchie-2019-111	Ritchie, Sandy and Sproat, Richard and Gorman, Kyle and van Esch, Daan and Schallhart, Christian and Bampounis, Nikos and Brard, Benoît and Fromseier Mortensen, Jonas and Holt, Millie and Mahon, Eoin	2019		111	questionnaire	English	Global	https://www.isca-speech.org/archive/Interspeech_2019/pdfs/2807.pdf	Ritchie2019		This list represents 111 numerals, used in the development of computational solutions to number verbalization in various languages. This initiative is part of a broader research effort at Google, enhancing the available machine learning techniques for building number names grammars across a wide range of languages for application in automatic speech recognition and text-to-speech synthesis systems.	3530-3534	
 Joo-2019-100	Joo, Ian	2019		100	ranked, basic	English	Global		Joo2019		This list of 100 items is based on the [Leipzig-Jakarta concept list](:ref:Tadmor-2009-100) and was employed in the study of phonosemantic patterns in the basic vocabulary of 66 genealogically unrelated languages.	1-12	
 Tolmie-1884-211	Tolmie, William F. and Dawson, George M.	1884		211	basic	English	Languages of the Native Tribes of British Columbia	https://open.library.ubc.ca/collections/bcbooks/items/1.0222394	Tolmie1884	Tolmie1884	This collection of 211 items, based on [Instructions for Research Relative to the Ethnology and Philology of America](:bib:Gibbs1863), was collected by the authors during expeditions in the British Columbia territory during the winters of 1875 and 1876. It contains basic vocabulary terms in addition to local flora and fauna terms. The core list of 211 items, also known as Dawson-Tolmie Vocabularies, is extended by appendices with extended vocabularies for specific languages in the area, but is not considered part of the main vocabulary list.		Tolmie-Dawson Vocabulary
-Winter-2016-300	Bodo Winter	2016		300	ranked	English	English	https://github.com/bodowinter/iconicity_senses/tree/master/data	Winter2016		This list of 300 items was used to elicit modality exclusivity norms of selected verb concepts, for which participants provided perceptual strength ratings across five sensory modalities (i.e., hearing, taste, touch, smell, and vision).		
+Winter-2016-300	Winter, Bodo	2016		300	ranked	English	English	https://github.com/bodowinter/iconicity_senses/tree/master/data	Winter2016		This list of 300 items was used to elicit modality exclusivity norms of selected verb concepts, for which participants provided perceptual strength ratings across five sensory modalities (i.e., hearing, taste, touch, smell, and vision).		
+Lynott-2013-400	Lynott, Dermot and Connell, Louise	2013		400	ranked	English	English	https://link.springer.com/article/10.3758/s13428-012-0267-0	Lynott2013		This list of 400 items was used to elicit modality exclusivity norms of selected noun concepts, for which participants provided perceptual strength ratings across five sensory modalities (i.e., hearing, taste, touch, smell, and vision).		
```

Comment:
Looks like you didn't pull up-to-date master branch after your previous PR has been merged?

Reply:
Doesn't seem to have done any harm, though. Looks fine otherwise!

Reply:
I did, but I've changed the order of the name because it didn't confirm to the "surname, name" order of the other entries.

Reply:
Ah! Didn't catch that, sorry, my bad. :)